lib/actions/sendTemplate.js

```diff
@@ -112,7 +105,7 @@ function getTemplateModel(cfg) {
         result.forEach((it) => {
             model[it.name] = it.name;
         });
-        console.log('Found %s templates', result.length);
+        logger.info('Found %s templates', result.length);
```

Comment:
Why logger instead of this.logger?

Reply:
It's metadata uploader. No difference here